.github/workflows/pypi-upload.yml

```diff
@@ -16,8 +17,23 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Autobump version
+        run: |
+          # from refs/tags/v1.2.3 get 1.2.3
+          VERSION=${GITHUB_REF/refs\/tags\//}
+          PLACEHOLDER='version="develop"'
```

Comment:
I see the develop line here. 🤔

Reply:
To also answer your other 2 questions about `develop`, it can be any string (or even be empty).
The magic in this workflow is actually happening in line#36. After stripping off the variables, it'll look something like

```
sed -i "s/develop/1.2.3/g" script.py
```

So we are replacing the string `develop` in that file with the tag(1.2.3) that we get from the GitHub context. This also means that we have no way to know what's the latest MS version from codebase without having to look at GitHub or PyPI.

Reply:
Cool, thanks for the explanation 👍